docs/source/quickstart.rst

```diff
@@ -1,9 +1,9 @@
 Quick start
 ===========
 
-Welcome to Ignite quick start guide that just gives essentials of getting a project up and running.
+Welcome to **PyTorch-Ignite** short tutorial, a quick start guide that just not only gives you the essentials of getting a project up but also importance of running through the code.
```

Comment:
Could you please rephrease this. Let's use : `**PyTorch-Ignite** quick start guide`.

> just not only gives you the essentials of getting a project up but also importance of running through the code.

Let's say what it is this quick guide instead of "not only this, but that".